connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -1092,7 +1092,7 @@ public void run() {
                                     return;
                                 }
                                 String reconfigUrl = RestServer.urlJoin(leaderUrl, "/connectors/" + connName + "/tasks");
-                                RestClient.httpRequest(reconfigUrl, "POST", rawTaskProps, null, config);
+                                RestClient.httpRequest(reconfigUrl, "POST", null, rawTaskProps, null, config);
```

Comment:
If the REST API requires authorization, won't this request fail due to lack of headers?

Reply:
Any answer to this?

Reply:
Hi. I don't think they we check headers during authorization for this!

Reply:
Hi @haidangdam, you mean there is no Authorization for POST /connectors/xx/tasks, so anyone can change tasks configuration ??

Reply:
Currently this endpoint should be secured through other means, such as mutual TLS.